### ISSUE TITLE
SysTraceWorker Assertion

### ISSUE THREAD

**Issue by HalfVoxel** [wolfpld/tracy#472]
When running, I reliably get this assertion after a few second or minutes of runtime (it's a bit random):
```
rust: tracy/client/TracySysTrace.cpp:1309: void tracy::SysTraceWorker(void*): Assertion `sel >= 0 || activeNum == 0' failed.
```

Using Tracy 0.8.1 via rust (`tracy-client = 0.13`).

**Comment by wolfpld** [comment]
This is fixed in 2240fd3b. Please do not use obsolete releases.

**Comment by HalfVoxel** [comment]
Ah. Thanks! Yeah, I normally wouldn't, but the `profiling` crate hasn't yet updated its dependencies.
Good to know it's fixed!
